zerver/openapi/zulip.yaml

```diff
@@ -184,6 +184,41 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/BadEventQueueIdError'
+    real-time:
```

Comment:
So, this is obviously a way to encode the data; do OpenAPI validators allow us making up an HTTP method like this?